Docker/amazon/amazon-1-vars.json

```diff
@@ -0,0 +1,10 @@
+{
+  "_comment": "Build with `packer build -var-file=amazon-1-vars.json amazon.json`",
+  "docker_image": "amazonlinux:2018.03",
```

Comment:
why not amazonlinux:1 ?

Reply:
This is the image we pull in.

Reply:
yeah, https://hub.docker.com/_/amazonlinux . 1 is easier to understand what is happening.